gasometer/src/lib.rs

```diff
@@ -50,6 +50,14 @@ impl<'config> Gasometer<'config> {
 		}
 	}
 
+	#[inline]
+	/// Reference of the inner gasometer data.
+	pub fn inner(
```

Comment:
I'd perfer we don't make `Inner` public. Would you mind to create those accessor functions in `Gasometer` that read in from `Inner` instead?